internal/config/io_test.go

```diff
@@ -86,7 +86,6 @@ mounts:
 			cfg: `root:
   autoclip: true
   autoimport: false
-  autosync: false
```

Comment:
Please, let all the tests as they are for now, since we have tests dedicated to test each previous config we ever had.
So, all changes to this file should be reverted.